aspnetcore/tutorials/Microsoft.dotnet-openapi.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Develop ASP.NET Core apps using OpenAPI
+author: ryanbrandenburg
+description: This tutorial demonstrates how to use the 'Microsoft.dotnet-openapi' tool to add references to OpenAPI files 
+ms.author: riande
+ms.date: 08/21/2019
+uid: tutorials/Microsoft.dotnet-openapi
+---
+# Develop ASP.NET Core apps using OpenAPI tools
+
+By Ryan Brandenburg
+
+`Microsoft.dotnet-openapi` is a tool for managing OpenAPI references within your project.
+
+## Installation
+
+To install the tool simply execute `dotnet tool install -g Microsoft.dotnet-openapi` from your terminal of choice.
+
+## Commands
+
+### Add Commands
+
+Adding an OpenAPI reference using any of the below commands results in an item similar to the below being added to your CSProj file:
+```xml
+<OpenApiReference Include="openapi.json" />
+```
+This reference, along with the code generator specific packages which were also added, will result in the generated client code being available for use by the rest of your code.
+
+<!-- TODO: Restore after https://github.com/aspnet/AspNetCore/issues/12738
```

Comment:
Not a big fan of checking in commented-out code

Reply:
It needs to be restored wholesale once project-references are restored to the OpenAPI world, so this seems to be the easiest way to set that up. It's not really different from how we've got the AddProjectReferenceCommand class that we just aren't referencing anymore.

Reply:
:ok: